web/routes.go

```diff
@@ -63,3 +70,12 @@ func initiateTranscriptionJobHandler(w http.ResponseWriter, r *http.Request) {
 func healthHandler(w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte("healthy!"))
 }
+
+// jobStatusHandler takes a POST request and returns status message.
```

Comment:
It actually takes a get request. I don't think you need to specify the type of request it takes here. Instead write a comment like

```go
// jobStatusHandler returns the status of a task with given id
```